css/ai.css

```diff
@@ -0,0 +1,142 @@
+body {
+    margin: 0;
```

Comment:
` margin: 0` is already set on the body here: https://github.com/ptcrealitylab/vuforia-spatial-toolbox-userinterface/blob/59137ea3a9fd546dd3074e018806c04f9c033824/css/index.css#L75